course_content/iris_course/solutions/iris_exercise_6.1d

```diff
@@ -0,0 +1,14 @@
+
+# Iris course sample answer 6.1d
+# changing cube metadata items
+
+e1_copy = e1.copy()
+e1_copy.rename('potential_temperature')
+print(e1_copy.summary(shorten=True))
+print("The cube name is simply changed to the requested name.")
+print("NOTE: however these aren't really potential temperatures at all, the values were absolute temperatures.")
+print('')
+e1_copy.units = 'days'
+print(e1_copy.summary(shorten=True))
+print("NOTE: 'potential_temperature' can't be measured in 'days' at all, but Iris takes no notice of this."
+      "\n  - ** HANDLE WITH CARE !!** ")
```

Comment:
I don't really like how messy these sample solutions are getting. I think it is making it harder to see the code, which is what the focus should be